lib/BillTechLinksManager.php

```diff
@@ -19,11 +19,11 @@ public function getCustomerPaymentLinks($customerId)
 	{
 		global $DB;
 		$rows = $DB->GetAll("select bpl.*, c.docid
-                                from billtech_payment_links bpl
-                                         left join cash c on c.id = bpl.src_cash_id
-                                         left join billtech_payments bp on bpl.token = bp.token
-                                where bp.id is null
-                                  and customer_id = ?", array($customerId));
+				from billtech_payment_links bpl
```

Comment:
Walczyłem tutaj z formatowanie dlatego tyle commitów, nic nie jest zmienione w tym zapytaniu, ale zamaist spacji są po prostu taby.

Reply:
Wg konwencji używamy spacji

Reply:
Tylko wszystkie pliki mają taby :p Dlatego chciałem już uspójnić, bo jak część jest tabów, a część spacji to najgorzej. Mogę w kolejnym PRze zrobić refactor z przerzuceniem wszystkiego na spacje.